benchmarks/cpp_benchmark.cc

```diff
@@ -121,9 +121,10 @@ class ParseNewArenaFixture : public Fixture {
   virtual void BenchmarkCase(benchmark::State& state) {
     WrappingCounter i(payloads_.size());
     size_t total = 0;
+    Arena arena;
 
     while (state.KeepRunning()) {
-      Arena arena;
+      arena.Reset();
```

Comment:
Does this make any difference in the benchmark result?

Reply:
Seems a little different for some messages.